website/docs/r/edgegateway_settings.html.markdown

```diff
@@ -0,0 +1,102 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_edgegateway_settings"
+sidebar_current: "docs-vcd-resource-edgegateway-settings"
+description: |-
+  Provides a vCloud Director edge gateway global settings. This can be used to update global edge gateways settings related to firewall and load balancing.
+---
+
+# vcd\_edgegateway\_settings
+
+Provides a resource that can update vCloud Director edge gateway global settings either as System Administrator or as
+Organization user.
+
+The main use case of this resource is to allow both providers and tenants to change edge gateways global settings (such as
+enabling load balancing or firewall) when the edge gateway was created outside of Terraform.
+A second use case is when the provider creates the edge gateway using Terraform, and then delegates the tenant to change
+some settings for further operations.
+
+~> **Warning:** The edge gateway settings info is tied to an edge gateway. Thus, there could be only one instance per 
+edge gateway. Using a different definition for the same edge gateway ID will result in a previous instance to be overwritten.
+
+!> **Warning:** Using a `vcd_edgegateway` and a `vcd_edgegateway_settings` for the same entity in the same configuration
+does not work correctly, as the main purpose of this resource is to handle general settings when the edge gateway was created outside of Terraform.
```

Comment:
After all the experimentation, this is more precise:

```suggestion
!> **Warning:** Using a `vcd_edgegateway` and a `vcd_edgegateway_settings` for the same entity is not supported, as the main purpose of this resource is to handle general settings when the edge gateway was created outside of Terraform.
```

Reply:
I liked more `not recommended` rather than `doesn't work` as you suggested. And I like even less `not supported`, because it is not true. I think `not recommended` is way better.

Reply:
OK, we can leave `doesn't work`, but let's remove `in the same configuration`, because that is not true.

Reply:
fixed